disaster-uploader.py

```diff
@@ -0,0 +1,338 @@
+from dotenv import load_dotenv
+import requests
+import json
+import pandas as pd
+import numpy as np
+import argparse
+import os
+from functools import partial
+
+load_dotenv()
+
+# Define CKAN API endpoint and your API key]
+fields = []
+csv_path =os.getenv('disaster_filename')
+csv_delimiter = ","
+ckan_api_url = os.getenv('ckan_url')
+api_key =  os.getenv('api_keys')
+resource_id = os.getenv('disaster_resource_id')
+
+def datastore_delete(ckan_api_url, resource_id, api_key):
+    print('Deleting datastore for ' + resource_id)
+
+    u = ckan_api_url + "/api/3/action/datastore_delete"
+    r = requests.post(u, headers={
+      "X-CKAN-API-Key": api_key,
+      "Accept": "application/json",
+      'Content-Type': 'application/json',
+    }, data=json.dumps({
+        "resource_id": resource_id,
+        "force": "true"
+    }))
+
+    return json.loads(r.text)
+
+def datastore_create(records, fields, ckan_api_url, resource_id, api_key):
```

Comment:
Method seems to be repeated between several files. Should be extracted to helper file and imported.